pinot-core/src/main/java/org/apache/pinot/core/query/reduce/GroupByDataTableReducer.java

```diff
@@ -136,23 +137,23 @@ private void reduceToResultTable(BrokerResponseNative brokerResponseNative, Data
     if (havingFilter != null) {
       HavingFilterHandler havingFilterHandler = new HavingFilterHandler(havingFilter, postAggregationHandler);
       while (rows.size() < limit && sortedIterator.hasNext()) {
-        Object[] row = sortedIterator.next().getValues();
-        extractFinalAggregationResults(row);
+        Object[] values = sortedIterator.next().getValues();
```

Comment:
(minor) Suggest reverting it. I feel `row` is more specific, `values` is too generic